modules/kernel/mkosi.build.chroot

```diff
@@ -101,14 +101,15 @@ fi
 
 KERNEL_RELEASE=$(make $EXTRA -s kernelrelease)
 
-ln -sf "$BUILDDIR" /tmp/lib
+mkdir -p /tmp/usr
+ln -sf "$BUILDDIR" /tmp/usr/lib
 mkdir -p "$BUILDDIR/modules/$KERNEL_RELEASE"
 find "$BUILDDIR/modules" -mindepth 1 -maxdepth 1 ! -name "$KERNEL_RELEASE" -type d -exec rm -r {} +
 make $EXTRA INSTALL_MOD_PATH=/tmp modules_install
```

Comment:
I think we'll need the same `MODLIB` change here as well?

Reply:
I forgot to commit it :sweat_smile:

Reply:
It does work on openSUSE without it, but I suspect other distros might not like it.